src/handlers/generate-erc721-permit.ts

```diff
@@ -102,12 +114,6 @@ export async function generateErc721PermitSignature(
     throw new Error("Provider is not defined");
   }
 
-  if (!_nftContractAddress) {
-    const errorMessage = "NFT contract address is not defined";
-    _logger.error(errorMessage);
-    throw new Error(errorMessage);
-  }
-
```

Comment:
This block of code was removed since above a validation is already performed when `NFT_CONTRACT_ADDRESS` is not defined or its string value is empty